websites/H/Holodex/dist/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+  "service": "Holodex",
+  "author": {
+    "name": "creeperkafasipw",
+    "id": "442685489833705483"
+  },
+  "category": "videos",
+  "logo": "https://i.imgur.com/ywSCOHi.png",
+  "thumbnail": "https://i.imgur.com/FM9qnIG.png",
+  "version": "1.2.0",
```

Comment:
```suggestion
  "version": "1.0.0",
```

Reply:
I was bumping up the version cuz of some bugfixes/edge-cases and some extra support but i guess the version only needs to be changed when i make a new pr?.. thanks for the info, i'll change the version back to 1.0.0

Reply:
> but i guess the version only needs to be changed when i make a new pr?

yup